test/pytest/test_bridge.py

```diff
@@ -161,6 +161,7 @@ async def assert_bus_notify(self, path: str, iface: str, expected: Dict[str, obj
         if bus is None:
             bus = await self.ensure_internal_bus()
         notify = await self.next_msg(bus)
+        assert 'notify' in notify, notify
```

Comment:
I don't understand the second "notify". Doesn't pytest already show the notify value on a failure? If not, then `self.assertIn('notify', notify)` may be cleaner?